hack/test-cmd.sh

```diff
@@ -308,7 +308,7 @@ osadm registry --create --credentials="${OPENSHIFTCONFIG}"
 # delete the registry resources
 osc delete dc docker-registry
 osc delete svc docker-registry
-osc delete rc docker-registry-1
+[ ! "$(osc get rc docker-registry-1)" ]
```

Comment:
Is there any way that this could flake, perhaps if there's a delay in the dc deletion cascading to delete the rc?